.gitignore

```diff
@@ -1 +1,2 @@
 node_modules
+dist
```

Comment:
@lottspot I recommend being more specific with .gitignore entries, as I've seen bare names like this become wildcards throwing wrenches into things. This line for example would exclude a file called "dist" several directories down. Use the entry `/dist/` instead which will only apply to a root directory named "dist"

Similarly I'd use `node_modules/` to ignore any directory name "node_modules"  at any depth. Probably less chance of accidents here than with a generic word like "dist", but being as specific as possible is a good practice to establish in .gitignore before you end up with a dozen bare words and wildcards